README.md

```diff
@@ -0,0 +1,30 @@
+# datarockets rails-composer
+
+## Install
+
+Install specific_install wich hel install forked gems:
+```
+gem install specific_install
+```
+
+Install gem:
+```
+gem specific_install -l 'https://github.com/datarockets/rails_apps_composer'
+```
+
+## Usage
+
+Creating new app:
+```
+rails_apps_composer new app_name
+```
+
+This is changed rails_apps_composer gem for datarockets team.
```

Comment:
@MProG what do you think about adding here those things that you've already added to the recipes? Plus in the future we'll just update README with new changes